### PR TITLE
Add copy_blobs to GCloudStorage

### PR DESCRIPTION
**Pull request by nradz**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 setuptools.setup(
     name='UDASwissKnife',
-    version='0.7.1',
+    version='0.8.0',
```
Comment (eoterogarcia) [inline setup.py:20]: 0.8.0 or 0.7.2? because what you add doesn't break all of the above to make it a major release, right?

**Review by eoterogarcia — APPROVED** [review]
Just a comment related with version number, and remember to add the release number in changelog before merging

File: setup.py (hunk 1 above)
Comment (nradz) [reply setup.py:20]: 0.8.0 because it is a new functionality. It would be 0.7.2 if it had be a bugfix.

**Review by ju4nlu — APPROVED** [review]

**Review by ju4nlu — COMMENTED** [review]
Very good

**Pull request merged**